handler/chapter.go

```diff
@@ -1,15 +1,28 @@
 package handler
 
 import (
-	r "scriptura/scriptura-api/repository"
+	"scriptura/scriptura-api/repository"
 	"scriptura/scriptura-api/utils"
 
 	"github.com/gofiber/fiber/v2"
 )
 
+// GetChapter
+//
+//	@Summary		Get chapter details
+//	@Description	Retrieve details of a chapter including book and verses based on input criteria.
+//	@Tags			Chapter
+//	@Accept			json
+//	@Produce		json
+//	@Param			id	path		int				true	"Chapter id"
```

Comment:
Here, the chapter param specifies the int type.

Reply:
Fixed